### PR TITLE
Dtspb 3298 make will lodgement cm optional

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-3298

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -462,12 +462,12 @@ pact {
 task buildBackOfficeXlsx(type: Exec) {
   environment 'CCD_DEF_CASE_SERVICE_BASE_URL', 'localhost:4104'
 
-  def input = project.file('ccdImports/configFiles/CCD_Probate_Backoffice')
+  def input = project.file('ccdImports/configFiles/CCD_Probate_Legacy_Search')
```
Comment (DelythJustice) [inline build.gradle]: what's this about?

File: build.gradle [hunk 2]
```diff
@@ -462,12 +462,12 @@ pact {
 task buildBackOfficeXlsx(type: Exec) {
   environment 'CCD_DEF_CASE_SERVICE_BASE_URL', 'localhost:4104'
 
-  def input = project.file('ccdImports/configFiles/CCD_Probate_Backoffice')
```
Comment (sanjayparekh-hmcts) [inline build.gradle]: We will revert the change on this file after QA

Comment (sanjayparekh-hmcts) [reply build.gradle]: Ive removed this - doesnt actually provide any real improvement

File: build.gradle (hunk 1 above)
Comment (sanjayparekh-hmcts) [reply build.gradle]: see above

**Review by DelythJustice — APPROVED** [review]
